### ISSUE TITLE
Deprecated use of numpy.matrix in pympler/asizeof.py

### ISSUE THREAD

**Issue by LucHermitte** [pympler/pympler#121]
I've an unit test that triggers the following message with numpy 1.19.5, and Python 3.6.9. 

```
myenv/lib/python3.6/site-packages/pympler/asizeof.py:1513
  myenv/lib/python3.6/site-packages/pympler/asizeof.py:1513: PendingDeprecationWarning: the matrix subclass is not the recommended way to represent matrices or deal with linear algebra (see https://docs.scipy.org/doc/numpy/user/numpy-for-matlab-users.html). Please adjust your code to use regular ndarray.
    numpy.matrix(range(0)), numpy.ma.masked_array([])):

-- Docs: https://docs.pytest.org/en/stable/warnings.html
```

**Comment by mrJean1** [comment]
A patch for `asizeof.py` has been submitted.

**Comment by cristianocca** [comment]
Can we get this merged? Seeing the same thing:

```
[2021-09-24 09:28:34] WARNING [py.warnings:109] env\lib\site-packages\pympler\asizeof.py:1513: PendingDeprecationWarning: the matrix subclass is not the recommended way to represent matrices or deal with linear algebra (see https://docs.scipy.org/doc/numpy/user/numpy-for-matlab-users.html). Please adjust your code to use regular ndarray.
  numpy.matrix(range(0)), numpy.ma.masked_array([])):
```

**Comment by Pankrat** [comment]
The fix is included in the 1.0 version that has just been pushed to PyPI. I hope this solves the issue.
